rootfs-builder/alpine/Dockerfile.in

```diff
@@ -0,0 +1,3 @@
+From golang:@GO_VERSION@-alpine3.7
```

Comment:
Should that be `-alpine@OS_VERSION@`?

Reply:
nop, this is the official golang image

Reply:
Sure, but it seems to reflect the current version of alpine: https://hub.docker.com/_/golang/.

Reply:
`OS_VERSION` is the OS version on created rootfs. We can stick with alpine3.7 to build it.